velox/functions/prestosql/aggregates/tests/MaxSizeForStatsTest.cpp

```diff
@@ -40,7 +40,7 @@ TEST_F(MaxSizeForStatsTest, nullValues) {
       makeNullableFlatVector<float>({std::nullopt, std::nullopt}),
       makeNullableFlatVector<double>({std::nullopt, std::nullopt}),
       makeNullableFlatVector<bool>({std::nullopt, std::nullopt}),
-      makeNullableFlatVector<Date>({std::nullopt, std::nullopt}),
+      makeNullableFlatVector<int32_t>({std::nullopt, std::nullopt}, DATE()),
```

Comment:
Remove this column. Since Date is no longer supported here we needn't test. Same for other functions in this file.